ui/app/components/allocation-status-bar.js

```diff
@@ -39,7 +39,8 @@ export default class AllocationStatusBar extends DistributionBar {
       'failedAllocs',
       'runningAllocs',
       'startingAllocs',
-      'lostAllocs'
+      'lostAllocs',
```

Comment:
Is this dependency missing on line 28?

Reply:
Weird -- yeah, looks like it is. Side-effect to this PR, but I'll add it in. In theory what would this have done, prevent auto-refresh of the chart when a new lost allocation showed up?